projects/demo/src/components/control/index.html

```diff
@@ -0,0 +1,36 @@
+<tr>
+    <td colspan="3">
+        <span tuiTitle>
+            <strong>ControlOptions</strong>
+            <span tuiSubtitle>Applied as a host directive</span>
+        </span>
+    </td>
+</tr>
+
+<tr
+    *ngIf="!hiddenOptions.includes('readonly')"
+    name="[readonly]"
+    tuiDocAPIItem
+    type="boolean"
+    [(value)]="readonly"
+>
+    Readonly state
+</tr>
+<tr
+    *ngIf="!hiddenOptions.includes('disabled')"
+    name="[disabled]"
+    tuiDocAPIItem
+    type="boolean"
+    [(value)]="disabled"
+>
+    Disabled state
```

Comment:
We should probably add a note to use `control.disable()/.enable()` for reactive forms like we had in old docs.

Reply:
> like we had in old docs

I can't find this notification in old docs. 
Will it be enough just to add text: "Use `control.disable()/.enable()` for reactive forms" ?

Reply:
![image](https://github.com/user-attachments/assets/b5c2bc22-d133-46db-9a55-71da74e56284)

You can write it as you see fit, we just want to make sure people understand this is a form thing, not our input, and for template driven form you're supposed to use `[disabled]` while for reactive forms you should call `control.disable()/.enable()` or create your own `[disabled]` directive that does so under the hood declaratively.

Reply:
Okay, I get it :)